docs/sdk/src/reference_docs/extrinsic_encoding.rs

```diff
@@ -56,7 +56,7 @@
 //!     version_and_signed,
 //!     from_address,
 //!     signature,
-//!     signed_extensions_extra,
+//!     transaction_extensions_extra,
 //! )
 //! ```
 //!
```

Comment:
we could also update the doc above and below. instead of one bit for the transaction type it is 2.

Reply:
Done.